server/src/test/java/io/crate/integrationtests/OpenCloseTableIntegrationTest.java

```diff
@@ -40,6 +45,160 @@ public void prepareClosedTable() {
         execute("alter table t close");
     }
 
+    @UseJdbc(1)
```

Comment:
Is the `UseJdbc(1)` required here?

Reply:
it is not needed, I used the wrong crate error code + HTTP code, so the assertion worked only for the pg but not for HTTP case.